shopgate/pwa
# Added rxjs operator `mergeMap` to the main stream

# Description

Added rxjs operator `mergeMap` to the main stream

## Type of change


## How to test it

Run the project locally with the classic extensions attached using the latest beta of the platform-sdk and start the `home` page. There should be no `mergeMap` error popping up anymore .
